frontend/views/pages/Home.vue

```diff
@@ -31,8 +31,10 @@ main.c-splash(data-test='homeLogo' v-if='!$store.state.currentGroupId')
   .create-or-join(v-else key='body-create')
     .card
       svg-create-group
-      h3.is-title-3 Create
-      p Create a new group and invite your friends.
+      h3.is-title-3
+        i18n Create
+      p
+        i18n Create a new group and invite your friends.
```

Comment:
For both of these it's preferable to keep the line count small by using `i18n.is-title-3(tag='h3')` and `i18n(tag='p')` respectively

Reply:
Ok, I tought less tokens was better, but the difference is small anyway